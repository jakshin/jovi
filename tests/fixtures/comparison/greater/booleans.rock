(Using the greater-than comparison operator on booleans)

Rightness is right
Wrongness is wrong

Say "booleans aren't greater than other booleans"
Rightness is right
Wrongness is wrong
say Rightness is greater than Rightness
say Wrongness is greater than Wrongness
say Rightness is greater than Wrongness
say Wrongness is greater than Rightness
say true is greater than true
say false is greater than false
say true is greater than false
say false is greater than true

(We test booleans against numbers and strings in numbers.rock and strings.rock)
