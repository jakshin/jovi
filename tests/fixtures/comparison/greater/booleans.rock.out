booleans aren't greater than other booleans
false
false
false
false
false
false
false
false
